slick/src/main/scala/akka/stream/alpakka/slick/javadsl/Slick.scala

```diff
@@ -221,4 +365,11 @@ object Slick {
   private def toDBIO[T](javaDml: JFunction[T, String]): T => DBIO[Int] = { t =>
     SQLActionBuilder(javaDml.asScala(t), SetParameter.SetUnit).asUpdate
   }
+
+  private def toDBIO[T](javaDml: JBiFunction[T, Connection, PreparedStatement]): T => DBIO[Int] = { t =>
+    SimpleJdbcAction { ctx =>
```

Comment:
I need help with this. I do not very know a Slick and I'm not sure about transactionality every `SimpleJdbcAction`.
It works in tests and our application, but I'm not sure that it's absolutely correct.

Reply:
I don't get your question about transactionality. 

This DBIO will have a single statement and will run in its own transaction and not participating to any other transaction. If the user had access to it they could `flatMap` with another DBIO. In that case, they will need to add `.transactionally` to wrap both DBIO on the same tx. 

Anyway, that's not applicable here. 

However, what could be an addition is a `BiFunction[T, Connection, List[PreparedStatement]]`. 

The user could then return a few prepared statements, all bound to the same connection. 

In such a case, we would convert each of them to a DBIO and indeed execute them in the same transaction using `.transactionally`.

Can be added on another PR though.

Reply:
> This DBIO will have a single statement and will run in its own transaction and not participating to any other transaction.

Exactly this is what I wanted to clarify. Thank you! 👍